src/platform/imx8/include/platform/lib/dma.h

```diff
@@ -18,6 +18,8 @@
 #define DMA_ID_EDMA0	0
 #define DMA_ID_HOST	1
 
+#define dma_chan_irq(dma, chan) (dma_irq(dma) + chan)
```

Comment:
Only added so it compiles? Because the per-channel IRQ numbers are rather weird on this platform.